tutorials/vue/nexttick.md

```diff
@@ -0,0 +1,27 @@
+The `nextTick()` function allows you to execute code after you have changed some data, and the virtual DOM has been updated by VueJS because of these data changes.
```

Comment:
"after you have changed some data and Vue has updated the virtual DOM, but **before** Vue renders the changes to the page."